docs/images/components.dot

```diff
@@ -0,0 +1,16 @@
+digraph components {
```

Comment:
What is this file?

Reply:
Sketch project files. Removing

Fixed in 45e1084

Reply:
It's ok to keep them as "source" of the current diagrams. I was just asking :)

Reply:
Did you finally re-added the files at the end? It's important to have the sources there in case somebody needs to modify in the future. Or in the case of .dot files are not needed for that purpose?